### DISCUSSION TITLE
How to use ChromeXt with Webview based browsers?

### DISCUSSION THREAD

**Discussion by ghost** [JingMatrix/ChromeXt#63] [Q&A] [answered]
I have LSPosed (Zygisk version) installed, but the websites to manage the scripts tell me I don't have ChromeXt installed, nor there's a userscripts related option in the browser.

Hopefully this could be fixed in a future release, take your time and thanks for this module and your efforts in making it

**Comment by JingMatrix** [comment]
It is working well on my phone.
Please install your ChromeXt using the APK provided by [Github Action](https://github.com/JingMatrix/ChromeXt/actions).

**Comment by ghost** [comment]
It's still not working, but could it be because I use Mulch as main Android System WebView through the OpenWebview Magisk module?

**Comment by JingMatrix** [comment]
Could you try to disable that module (restart your phone of course), and then tell me what happens?

In case you have useful logs, of LSPosed or of `adb logcat -s ChromeXt`, post it here.

**Comment by ghost** [comment]
Even with disabled OpenWebView module, the result is the same, I recorded the logs here from LSPosed app

https://drive.google.com/file/d/1z1ZtHf5E2Jn-eqCoTRYGXCqI0kJztpA0/view?usp=drivesdk

**Comment by JingMatrix** [comment]
No idea of what is happening.
Do you know to how to use `adb` tools for debugging?
If you wish to help more, you can search on Google how to use it and then post the output of command `adb logcat -s ChromeXt`.

Currently, I cannot reproduce your case. By the way, I tested it with `Midori Lite` 1.7.

**Comment by JingMatrix** [comment]
I am guessing the possible issue.
Could you try the latest build and tell me if your problem is solved?

**Comment by ghost** [comment]
Thanks for trying, but still the module is not working, according to the websites.

Here the new logs, I just have 2 modules installed in LSPosed, ChromeXt and Iconify (device looks customization, not related to browsers)

https://drive.google.com/file/d/1zOCPOJdmz6oqV854AK7l3XIKQszmIZ3j/view?usp=drivesdk

**Comment by JingMatrix** [comment, marked as answer]
I get the problem.
You should enable the module for `Midori` (only this is needed), nor for webview apps.
Enjoy it.
